src/pal/src/include/pal/context.h

```diff
@@ -375,6 +375,59 @@ const fpsimd_context* GetConstNativeSigSimdContext(const native_context_t *mc)
 #define MCREG_Pc(mc)        ((mc).arm_pc)
 #define MCREG_Cpsr(mc)      ((mc).arm_cpsr)
 
+
+// Flatterned layout of the arm kernel struct vfp_sigframe
+struct VfpSigFrame
+{
+    DWORD   magic;
+    DWORD   size;
+    DWORD64 D[32];
+    DWORD   Fpscr;
+    DWORD   Padding;
+    DWORD   Fpexc;
+    DWORD   Fpinst;
+    DWORD   Fpinst2;
+    DWORD   Padding2;
+};
+
+inline
+VfpSigFrame* GetNativeSigSimdContext(native_context_t *mc)
+{
+    size_t size = 0;
+
+    const DWORD VfpMagic = 0x56465001; // VFP_MAGIC from arm kernel
```

Comment:
I couldn't find a definition for `VFP_MAGIC` or `vfp_sigframe` in the arm root file system. 

- I chose to use different type and define names in case they are added to the arm userspace include in the future.

- I flattened since it made the code simpler

- I added the size assert to be exact to make sure my size was correct.

- I added padding members rather than using `__attribute(aligned...`.  Packed and aligned 8 structure is now the same.